src/python/pants/help/help_info_extracter.py

```diff
@@ -92,6 +107,16 @@ def compute_metavar(kwargs):
 
     return metavar
 
+  @staticmethod
+  def compute_choices(kwargs):
```

Comment:
Would be great to add the return type for this function. Don't worry about typing `kwargs`, though.

```suggestion
  def compute_choices(kwargs) -> Optional[str]:
```

Reply:
Got it, thanks!